AISKULight/src/index.ts

```diff
@@ -54,6 +54,7 @@ export class ApplicationInsights {
             _initialize();
           
             _self.initialize = _initialize;
+            _self.track = _track;
         
             proxyFunctions(_self, core, [
                 "track",
```

Comment:
You should remove this as this (might) "replace" the track you assigned on line 57.

Reply:
just about to move the proxy!